loadData.sh

```diff
@@ -47,3 +53,10 @@ fi
 # Submit data
 loadData $STATIC_DATA_DIR
 loadData $BUILD_DATA_DIR
+
+# Warm Data
+log "Warming indices..."
+warm findPrefsSafeByGpiiKey
```

Comment:
I don't think it is a good idea to have this hardcoded here. Views list can be extracted from ` GET /_design/views/` query.

Reply:
Great point! I've added some new code to query the list of views during the run and decide what to query based off of that.